pkg/kubelet/apis/config/types.go

```diff
@@ -369,10 +369,10 @@ type KubeletConfiguration struct {
 	// See http://kubernetes.io/docs/user-guide/compute-resources for more detail.
 	KubeReserved map[string]string
 	// This flag helps kubelet identify absolute name of top level cgroup used to enforce `SystemReserved` compute resource reservation for OS system daemons.
-	// Refer to [Node Allocatable](https://git.k8s.io/community/contributors/design-proposals/node/node-allocatable.md) doc for more information.
```

Comment:
link all the node-allocatable fields to https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/#node-allocatable instead?